src/reporters/aladdin/README.md

```diff
@@ -0,0 +1 @@
+# The Aladdin reporter
```

Comment:
@aladdin-add the main purpose with this reporter is that it should "translate" all diagnostics to chinese and output only in chinese.